index.bs

```diff
@@ -2330,22 +2336,24 @@ The <dfn method for=IDBFactory>databases()</dfn> method steps are:
 
 1. Run these steps [=in parallel=]:
 
-    1. Let |databases| be the [=/set=] of [=/databases=] in |storageKey|.
-        If this cannot be determined for any reason, then [=/reject=] |p| with
-        an appropriate error (e.g. an "{{UnknownError}}" {{DOMException}})
-        and terminate these steps.
+    1. [=Queue a database task=] to run these steps:
```

Comment:
I think it would be more consistent with the rest of the spec to not make all of the substeps queued on the database task source. The spec generally uses "in parallel" to mean _"post this over to some implementation-defined thread or process that's interacting directly with the database"_ and "queue a task" to mean _"post this over to the JS thread"_. So only the terminal steps of the algorithm (rejecting, resolving) should get queued.

So change the prior text to read _"If this cannot be determined for any reason, then queue a database to reject p with..."_ and don't update the indentation of the substeps. And then...